Source/FieldSolver/SpectralSolver/SpectralAlgorithms/PsatdAlgorithmRZ.cpp

```diff
@@ -85,17 +99,17 @@ PsatdAlgorithmRZ::pushSpectralFields(SpectralFieldDataRZ & f)
         {
 
             // All of the fields of each mode are grouped together
-            auto const Ep_m = Idx.Ex + Idx.n_fields*mode;
-            auto const Em_m = Idx.Ey + Idx.n_fields*mode;
-            auto const Ez_m = Idx.Ez + Idx.n_fields*mode;
-            auto const Bp_m = Idx.Bx + Idx.n_fields*mode;
-            auto const Bm_m = Idx.By + Idx.n_fields*mode;
-            auto const Bz_m = Idx.Bz + Idx.n_fields*mode;
-            auto const Jp_m = Idx.Jx + Idx.n_fields*mode;
-            auto const Jm_m = Idx.Jy + Idx.n_fields*mode;
-            auto const Jz_m = Idx.Jz + Idx.n_fields*mode;
-            auto const rho_old_m = Idx.rho_old + Idx.n_fields*mode;
-            auto const rho_new_m = Idx.rho_new + Idx.n_fields*mode;
+            int const Ep_m = Idx.Ex + Idx.n_fields*mode;
+            int const Em_m = Idx.Ey + Idx.n_fields*mode;
+            int const Ez_m = Idx.Ez + Idx.n_fields*mode;
+            int const Bp_m = Idx.Bx + Idx.n_fields*mode;
+            int const Bm_m = Idx.By + Idx.n_fields*mode;
+            int const Bz_m = Idx.Bz + Idx.n_fields*mode;
+            int const Jp_m = Idx.Jx + Idx.n_fields*mode;
+            int const Jm_m = Idx.Jy + Idx.n_fields*mode;
+            int const Jz_m = Idx.Jz + Idx.n_fields*mode;
+            int const rho_old_m = Idx.rho_old + Idx.n_fields*mode;
+            int const rho_new_m = Idx.rho_new + Idx.n_fields*mode;
```

Comment:
I changed `auto` to `int` here just because I thought it might be a bit easier for someone who goes through the code to learn immediately what these variables are (simply integer indices).